portfolio/src/main/webapp/script.js

```diff
@@ -27,18 +27,37 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
-/**
- * Adds a random fact about me to the page.
+/*
+ * Add a random fact about me to the about section of main page.
  */
+let randomFact = 0;
 function addRandomFact() {
   const factoids =
-      ['I\'ve never been stung by a bee 🐝', 'I love choir! I sing Alto 🎶', 'My dream vacation is Bali 🌴', 'I used to run a book blog! 📚',
-       'My favorite color is purple 💜' , 'My favorite composer is Debussy 🎹'];
+    ['I\'ve never been stung by a bee 🐝', 'I love choir! I sing Alto 🎶', 'My dream vacation is Bali 🌴', 'I used to run a book blog! 📚',
+     'My favorite color is purple 💜' , 'My favorite composer is Debussy 🎹'];
 
   // Pick a random fact.
-  const factoid = factoids[Math.floor(Math.random() * factoids.length)];
+  const index = Math.floor(Math.random() * factoids.length);
+
+  // Get a different index than last time.
+  if (index == randomFact) {
```

Comment:
This works too! Mostly my logic was meant to note that if you had an array of length n, you could just add any number between [1, n-1] to it to make it a new fact, and I'm also realizing my solution didn't even do that, it was able to accidentally add 0 anyways :)